### ISSUE TITLE
Could it possiable training multi class at same time?

### ISSUE THREAD

**Issue by lucasjinreal** [overfitover/pixor_pytorch#3]
Seems the loss only using binary_crossentropy. Could it possible training multi class at same time?

**Comment by overfitover** [comment]
There are a lot of things to improve,you can modify and commit,or post it yourself. you can do better. @jinfagang

**Comment by lucasjinreal** [comment]
@overfitover  We discuss in community:  [here](http://talk.strangeai.pro/category/27/3d%E7%9B%AE%E6%A0%87%E6%A3%80%E6%B5%8B)
